source/_static/js/pid-tune/1-base-sim.js

```diff
@@ -0,0 +1,180 @@
+class BaseSim {
+
+    constructor(div_id_prefix, stateUnits) {
+        this.speedGraph = null;
+        this.voltsGraph = null;
+        this.containerDiv  = document.getElementById(div_id_prefix + "_container");
+        var plotDrawDivVals = document.getElementById(div_id_prefix + "_plotVals");
+        var plotDrawDivVolts = document.getElementById(div_id_prefix + "_plotVolts");
+
+        this.valsChart = new Highcharts.Chart(plotDrawDivVals, dflt_options);
+        this.voltsChart = new Highcharts.Chart(plotDrawDivVolts, dflt_options);
+
+        this.voltsChart.addSeries({name: "Control Effort", color: '#00BB00'});
+        this.valsChart.addSeries({name: "Output", color: '#FF0000'});
+        this.valsChart.addSeries({name: "Setpoint", color: '#0000FF'});
+
+        this.voltsChart.xAxis[0].addPlotLine({color: '#BBBB00',width: 2, value: 0.0, id:"curTime"})
+        this.valsChart.xAxis[0].addPlotLine({color: '#BBBB00',width: 2,value: 0.0, id:"curTime"})
+
+        this.voltsChart.yAxis[0].setTitle({ text : "Volts"});
+        this.valsChart.yAxis[0].setTitle({ text :stateUnits });

```

Comment:
Plot y-axes should have a fixed scale depending on the mechanism being simulated, rather than being inferred from the calculated output values.